js/artFeature.js

```diff
@@ -15,5 +15,5 @@ const creatingArtFromHtml = () => {
 };
 
 document
-  .querySelector('html-code')
```

Comment:
Appreciate your patience.

Could you please see how can we utilize `querySelector` itself instead of changing into `getElementById`
https://dev.to/eidorianavi/queryselector-vs-getelementbyid-gm1
One of the available resource
I would still recommend to go through MDN docs though